pkg/app/api/grpcapi/web_api.go

```diff
@@ -563,6 +563,25 @@ func (a *WebAPI) GetPiped(ctx context.Context, req *webservice.GetPipedRequest)
 	}, nil
 }
 
+func (a *WebAPI) UpdatePipedDesiredVersion(ctx context.Context, req *webservice.UpdatePipedDesiredVersionRequest) (*webservice.UpdatePipedDesiredVersionResponse, error) {
+	if req.Version != "" && !strings.HasPrefix(req.Version, "v") {
+		return nil, status.Error(codes.InvalidArgument, "Version must be prefixed with a `v` character")
+	}
```

Comment:
Looks like protobuf also doesn't have any validation if it's empty. I agree with that Piped has an empty `DesiredVersion` but for this RPC, I'm guessing it shouldn't be empty, but what do you think?

Reply:
I was thinking the same firstly and had validation for that field (https://github.com/pipe-cd/pipe/pull/2553/commits/c89c49f4e4b32231d222366883838aad82e39f4e)
but after second thought empty value will allow users to disable remote upgrading if they want.
For sure, that is just a rare case so no strong opinion for it.

Reply:
Rethink about it, let me revert the latest commit to prevent an empty version. That makes thing becomes simpler.

Reply:
Thank you. Fixed it! https://github.com/pipe-cd/pipe/pull/2553/commits/1ae8bbc19d29b9b213e7d8519578019aa4f1a2fc

Reply:
Nice. As you said, the disabling feature could help users in various cases, so let's support it when necessary, not in a way other than specifying an empty character.